JS/Lesson34/goksel/recipe-app/src/modules/recipes/RecipeList/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Link } from 'react-router-dom';
+import { CardActionArea, CardContent, CardMedia, Grid, Typography } from '@mui/material';
+import Card from '@mui/material/Card';
+
+import "./styles.css"
+import { Recipe } from '../models';
+import { useDispatch, useSelector } from 'react-redux';
+
+
+
+export const RecipeList = () => {
+const dispatch = useDispatch();
+const recipes = useSelector((state: {recipes:Recipe[]})=> state.recipes)
```

Comment:
It's common to add something like this to the slice file: `export const selectorRecipes = (state) => state.recipes;`, then you can use it here like this:
`const recipes = useSelector(selectorRecipes);`.

Reply:
Fixed